ports/xbyak/portfile.cmake

```
@@ -1,14 +1,23 @@
 vcpkg_from_github(

```

```suggestion
vcpkg_fail_port_install(ON_ARCH "arm" ON_TARGET "uwp")

vcpkg_from_github(
```
Since it doesn't support arm and uwp, it would be better to add this supported message here.